assets/grafana/celeborn-dashboard.json

```diff
@@ -1357,34 +1357,19 @@
           ],
           "title": "metrics_ShutdownWorkerCount_Value",
           "type": "timeseries"
-        }
```

Comment:
Have you ever verified the dashboard?

Reply:
![SeaTalk_IMG_20240606_111048](https://github.com/apache/celeborn/assets/31424839/dec22801-464f-45b3-add7-eeb572d22e23)
![SeaTalk_IMG_20240606_111302](https://github.com/apache/celeborn/assets/31424839/d0b1a68b-e12c-4225-8812-1612183009f0)

Reply:
The monitoring of the panel is normal, but I found that there is a problem with the layout. This problem also existed before this PR. I think we may need to re-export the entire dashboard. For example, when I click the overall title, part of the overall panel will pop up. Go to the top of the dashboard.

Reply:
When i import
![WechatIMG84](https://github.com/apache/celeborn/assets/31424839/6fb1928a-c6a4-4164-a9e4-78bac86319cf)
Click overall
![WechatIMG85](https://github.com/apache/celeborn/assets/31424839/35fc5c4c-9dc6-496e-be8c-683db33da209)
Click Master
![SeaTalk_IMG_20240606_112033](https://github.com/apache/celeborn/assets/31424839/6a082672-ee10-430f-a093-a3d5748402ba)